deploy-cloudgov-docker.sh

```diff
@@ -64,10 +68,10 @@ update_backend()
 		# Do a zero downtime deploy.  This requires enough memory for
 		# two apps to exist in the org/space at one time.
 		#The `--var` parameter ingest a value into the ((docker-backend)) environment variable in the manifest.yml**
-		cf push $CGHOSTNAME_BACKEND --no-route -f tdrs-backend/manifest.yml --var docker-backend=$DOCKER_IMAGE_BACKEND --strategy rolling || exit 1
+		cf push $CGHOSTNAME_BACKEND --no-route -f tdrs-backend/manifest.yml --var docker-backend=$DOCKER_IMAGE_BACKEND  --var cf-space=$CF_SPACE --strategy rolling || exit 1
```

Comment:
The backend manifest now accepts the CF_SPACE as a variable to allow for injecting the route name in to the AV_SCAN_URL environment variable.